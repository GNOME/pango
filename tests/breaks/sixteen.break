hy‧phen|ation over­load
