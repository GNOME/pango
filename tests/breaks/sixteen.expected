Text:         ⁦h⁩ ⁦y⁩ ⁦‧⁩  ⁦p⁩ ⁦h⁩ ⁦e⁩ ⁦n⁩  ⁦|⁩  ⁦a⁩ ⁦t⁩ ⁦i⁩ ⁦o⁩ ⁦n⁩  [ ]  ⁦o⁩ ⁦v⁩ ⁦e⁩ ⁦r⁩ [0xad]  ⁦l⁩ ⁦o⁩ ⁦a⁩ ⁦d⁩  [0x0a]  
Breaks:     c  c c lc c c c c  lc c c c c c    lc c c c c      lc c c c c       Lc
Whitespace:                               x                             w       w 
Sentences:  bs                                                          e       b 
Words:      bs   e s        be bs         be   bs                       be      b 
Graphemes:  b  b b b  b b b b  b  b b b b b    b  b b b b      b  b b b b       b 
Hyphens:       i   ri i i i       i i i i         i i i i      i  i i i           
